api/api_test.go

```diff
@@ -0,0 +1,32 @@
+package api
+
+import (
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+)
+
+const TestAppPort = 23455
+
+func createServer(ch ClusterHandler) *http.Server {
+	addr := fmt.Sprintf(":%d", TestAppPort)
+
+	router := http.NewServeMux()
+	router.HandleFunc("/hello", Hello)
+	router.HandleFunc("/createservice", ch.CreateService)
+	router.HandleFunc("/listcluster", ch.ListCluster)
+	router.HandleFunc("/cluster", ch.GetCluster)
```

Comment:
This is only a subset of endpoints - but we can put everything here. Idea is to reuse this for actual integration tests.